README.md

```diff
@@ -6,6 +6,28 @@ This project aims to make journalists' experience working with SecureDrop less o
 
 This project is under active development, has known bugs and shortcomings, and is not ready for end users. This README is geared towards interested technical users and developers.
 
+### Detailed rationale
+
+SecureDrop's security model is proven, but it has major drawbacks:
```

Comment:
Perhaps link to https://docs.securedrop.org/en/stable/threat_model/threat_model.html ? It's a bit dated, but it's what we have documented public. As a newcomer, I'd wonder immediately what we mean by "SecureDrop's security model," and that's the best answer available in public docs.